core-java-modules/core-java-arrays-multidimensional/src/main/java/com/baeldung/array/reversearray/ReverseArrayElements.java

```diff
@@ -0,0 +1,89 @@
+package com.baeldung.array.reversearray;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Deque;
+import java.util.List;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+public class ReverseArrayElements {
+
+    public static void reverseRowsUsingSimpleForLoops(int[][] array) {
+        for (int row = 0; row < array.length; row++) {
+            for (int col = 0; col < array[row].length / 2; col++) {
+                int current = array[row][col];
+                array[row][col] = array[row][array[row].length - col - 1];
+                array[row][array[row].length - col - 1] = current;
+            }
+        }
+    }
+
+    public static void reverseRowsUsingNestedIntStreams(int[][] array) {
+        IntStream.range(0, array.length)
+            .forEach(row -> IntStream.range(0, array[row].length / 2)
+                .forEach(col -> {
+                    int current = array[row][col];
+                    array[row][col] = array[row][array[row].length - col - 1];
+                    array[row][array[row].length - col - 1] = current;
+                }));
+    }
+
+    public static void reverseRowsUsingCollectionsReverse(int[][] array) {
+        for (int row = 0; row < array.length; row++) {
+            List<Integer> collectionBoxedRow = Arrays.stream(array[row])
+                .boxed()
+                .collect(Collectors.toList());
+
+            Collections.reverse(collectionBoxedRow);
+
+            array[row] = collectionBoxedRow.stream()
+                .mapToInt(Integer::intValue)
+                .toArray();
+        }
+    }
+
+    public static void reverseRowsUsingCollectionsReverse(List<List<Integer>> array) {
+        array.forEach(Collections::reverse);
+    }
+
+    public static void reverseRowsUsingArrayUtilsReverse(int[][] array) {
+        for (int i = 0; i < array.length; i++) {
+            ArrayUtils.reverse(array[i]);
+        }
```

Comment:
We can simplify using enhanced loop:
```suggestion
        for (int[] ints : array) {
            ArrayUtils.reverse(ints);
        }
```